### PR TITLE
fixed bug with Bone.rotate, Bone.Scale (initialized private vars) -ad…

### PR DESCRIPTION
**Pull request by abow**
…ded Bone.setYawPitchRoll -added Bone.getAbsolutePosition

http://www.babylonjs-playground.com/#I6RJJ#13
http://www.babylonjs-playground.com/#11BH6Z#168

### CODE REVIEW

**Comment by abow** [comment]
Sorry don't merge yet.  I need to make one more commit.

**Comment by abow** [comment]
I think I'm all set.  Thanks

**Comment by abow** [comment]
I think I to a little overzealous.  This seems to have broken rotation in World Space.  :(

File: src/Bones/babylon.bone.ts [hunk 1]
```diff
@@ -329,5 +355,51 @@
...
+                tmat = this.getAbsoluteTransform();
+            }
+
+            result.x = tmat.m[12];
+            result.y = tmat.m[13];
+            result.z = tmat.m[14];
+
+        }
+
+        public static computeAbsoluteTransforms (bone:BABYLON.Bone): void {
```
Comment (deltakosh) [inline src/Bones/babylon.bone.ts]: Why static?

Comment (abow) [inline src/Bones/babylon.bone.ts]: because it's a recursive function and seemed to make more sense to me.

Comment (abow) [inline src/Bones/babylon.bone.ts]: Now that you mentioned it and I looked at it some more, I think it would be better as a public method.

**Comment by abow** [comment]
It looks like rotating in world space is working for Bone.rotate, but not for Bone.setYawPitchRoll.

**Comment by deltakosh** [comment]
Ok I'll wait to merge (thanks for the static stuff btw)

**Comment by abow** [comment]
Darn, I committed the js file.

**Comment by abow** [comment]
I discovered a bug that I will be committing a fix for in the next few minutes.

**Comment by abow** [comment]
I fixed that last bug.

http://www.babylonjs-playground.com/#I6RJJ#22

**Comment by abow** [comment]
Can you suggest a way to get the worldMatrix of the mesh from a bone so that we don't have to pass rotate and setYawPitchRoll the mesh when working in world space?

**Comment by deltakosh** [comment]
Bones are not linked to a specific mesh (as they can be applied to several meshes)

**Pull request merged**